svgdigitizer/electrochemistry/cv.py

```diff
@@ -1,24 +1,21 @@
-import yaml
 import pandas as pd
 import matplotlib.pyplot as plt
 from pathlib import Path
-from svgdigitizer.svgplot import SVGPlot
 
-class CV(SVGPlot):
-    def __init__(self, filename, sampling_interval=None):
-        self.svgfile = filename + '.svg' # Minidom parse does not accept a Path?
-        self.yamlfile = Path(filename).with_suffix('.yaml')
-        self.csvfile = Path(filename).with_suffix('.csv')
-        
+class CV():
+    def __init__(self, metadata, SVGPlot):
```

Comment:
I would not call this `SVGPlot`. Parameters are usually lowercase, i.e., `svgplot`.